target 'generate' fails on windows system due to backslashes in plugin.go

Running the `make` in `$GOPATH/src/github.com/mitchellh/packer` to run tests and build the packer binary will throw an error on a windows system `cmd.exe` as the generated  [`command\plugin.go`](https://gist.github.com/metalmolly/9da6b850c5cd1949e272) contains backslashes.

    C:\Users\MarkS\go\src\github.com\mitchellh\packer>make
    go get github.com/mitchellh/gox
    go get golang.org/x/tools/cmd/stringer
    go get golang.org/x/tools/cmd/vet
    go generate .
    2016/02/11 14:25:03 Generated command/plugin.go
    go fmt command/plugin.go
    can't load package:
    command\plugin.go:16:66: unknown escape sequence
    command\plugin.go:17:63: unknown escape sequence
    command\plugin.go:18:68: unknown escape sequence
    command\plugin.go:19:26: invalid import path: "github.com/mitchellh/packer/provisioner\ansible-local"
    command\plugin.go:20:21: invalid import path: "github.com/mitchellh/packer/provisioner\ansible"
    command\plugin.go:21:24: invalid import path: "github.com/mitchellh/packer/post-processor\artifice"
    command\plugin.go:22:21: invalid import path: "github.com/mitchellh/packer/post-processor\atlas"
    command\plugin.go:23:65: unknown escape sequence
    command\plugin.go:24:63: unknown escape sequence
    command\plugin.go:25:68: unknown escape sequence
    command\plugin.go:26:59: unknown escape sequence
    command\plugin.go:27:53: unknown escape sequence
    command\plugin.go:28:72: unknown escape sequence
    command\plugin.go:29:70: unknown escape sequence
    command\plugin.go:30:70: unknown escape sequence
    command\plugin.go:31:69: unknown escape sequence
    Makefile:39: recipe for target 'generate' failed
    make: *** [generate] Error 1

Using Go 1.5.3 on Windows 8 with Git 2.7.1 and Cygwin64 on latest commit [`54f8bc5`](https://github.com/mitchellh/packer/commit/54f8bc5ff4ed7ce8eb3a38e0975a0c2e284af96d).
A workaround is to comment the `go generate .` command after the first generation

--- a/scripts/generate-plugins.go
+++ b/scripts/generate-plugins.go
@@ -96,15 +96,15 @@ func makeImports(builders, provisioners, postProcessors []plugin) string {
 	plugins := []string{}
 
 	for _, builder := range builders {
-		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", builder.ImportName, builder.Path))
+		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", builder.ImportName, filepath.ToSlash(builder.Path)))
 	}
 
 	for _, provisioner := range provisioners {
-		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", provisioner.ImportName, provisioner.Path))
+		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", provisioner.ImportName, filepath.ToSlash(provisioner.Path)))
 	}
 
 	for _, postProcessor := range postProcessors {
-		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", postProcessor.ImportName, postProcessor.Path))
+		plugins = append(plugins, fmt.Sprintf("\t%s \"github.com/mitchellh/packer/%s\"\n", postProcessor.ImportName, filepath.ToSlash(postProcessor.Path)))
 	}
 
 	// Make things pretty

